protoc-gen-kotlin/lib/src/commonMain/kotlin/pbandk/gen/CodeGenerator.kt

```diff
@@ -118,38 +162,115 @@ open class CodeGenerator(
         // Messages can have circular references to each other (e.g. `pbandk.wkt.Struct` includes a `pbandk.wkt.Value`
         // field, but `pbandk.wkt.Value` includes a `pbandk.wkt.Struct` field). On Kotlin/Native the companion object
         // (e.g. `pbandk.wkt.Value.Companion`) is automatically frozen because it's a singleton. But Kotlin/Native
+
+        val allFields = type.sortedStandardFieldsWithOneOfs()
+        val chunkSize = 200
+        val needToChunk = allFields.size > chunkSize
+
         // doesn't allow cyclic frozen structures:
```

Comment:
Looks like this long comment block got split up in mid-sentence accidentallly. The entire comment block needs to be below these 3 variable declarations.

Reply:
Wow, another unacceptable thing on my behave. Sorry about this.